common/chains/client/chain_rpc_client.go

```diff
@@ -0,0 +1,22 @@
+package client
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink/v2/common/types"
+)
+
+type ChainRPCClient[
```

Comment:
This doesn't feel like a ChainRPCClient anymore, but rather like a node client API.

Reply:
True, just changed this.